### PR TITLE
Define reference scenarios and initiate benchmark of our different approaches

### PR DESCRIPTION
**Pull request by bowni**
- [x] Definition of scenarios and schemas (Eric): [link to editable schema](https://docs.google.com/drawings/d/1_ovwpBTx_Rno5FO1AcODM7cJ6a_EPaBOh9OYl952J-4/edit?usp=sharing)
- [x] Select an "open science" web tool for committing benchmark results (Nathanaël)
- [x] Highlight corrupted data more clearly
- [ ] Run each scenario at least once (Eric + help?)
- [x] Describe reference scenarios in the documentation (Eric)

### CODE REVIEW

**Comment by bowni** [comment]
@RomainGoussault @JustineBoulant with the Flexible Splitter PR #320 ready (pending reviews 😎), and the scenarios defined, I think we are all set to run them at least once. Let me know if you try some of them!
I am going to try "The Trio" on CIFAR10 :)

**Comment by bowni** [comment]
Hi everyone, as mentioned this morning in the bootcamp launch I suggest to merge this PR to include the definition of the reference scenarios.
We can then open another one in draft status, to add incrementally the first results we get on the reference scenarios.
So I put it in review now!

**Review by natct10 — APPROVED** [review]
+1

File: README.md [hunk 1]
```diff
@@ -171,6 +172,30 @@ Under `scenario_params_list`, enter a list of sets of scenario(s). Each set star
...
+
+In brief:
+
+- Scenarios 1 and 2 with 2 partners only enable simple baselines, with different data splits (each partner having samples of different classes in the first scenario, of all classes both in the second one), and introducing corrupted data;
+
+- Scenario 3 proposes a more realistic configuration, with partners having both samples of common classes and samples of different classes each;
+
+- Scenario 4 offers a case with 5 partners and an identical distribution of data samples of all classes, but with 1 partner having its data entirely corrupted;
+
+- Scenario 5 is more complex, with 11 partners, and a mix of identical distribution for some partners, different one for others, and corrupted data also.
```
Comment (arthurPignet) [inline README.md:193]: I think this description is unclear. 
Maybe : the samples of each class are distributed among various subset of partners, of which some overlap. 
Not sure that easier, but I think something is missing in the reference to identical distribution/different one

**Review by arthurPignet — APPROVED** [review]

**Comment by arthurPignet** [comment]
what do you think about adding a reference to the dataset(s) we plan to use ?

**Comment by bowni** [comment]
> what do you think about adding a reference to the dataset(s) we plan to use ?

File: README.md (hunk 1 above)
Comment (bowni) [reply README.md]: OK, clarified now

**Comment by bowni** [comment]
> what do you think about adding a reference to the dataset(s) we plan to use ?

I don't think this is where the focus should be. If needed, this is mentioned on the schema.

**Pull request merged**